curriculum/challenges/english/25-front-end-development/quiz-javascript-comparisons-and-conditionals/66edc47c11492ac5cf258ad9.md

```diff
@@ -17,439 +17,501 @@ Answer all of the questions below correctly to pass the quiz.
+"false"
 
 ---
 
-Placeholder distractor 2
+13
 
 ---
 
-Placeholder distractor 3
+-1
 
 #### --answer--
 
-Placeholder answer
+0
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What's the difference between `undefined` and `null`?
 
 #### --distractors--
 
-Placeholder distractor 1
+No difference.
 
 ---
 
-Placeholder distractor 2
+`null` automatically declared. (When there's no value)
 
 ---
 
-Placeholder distractor 3
+`undefined` means a variable is deliberately empty, while `null` is declared, but unassigned.
 
 #### --answer--
 
-Placeholder answer
+`undefined` means a variable is declared but unassigned, while `null` is absence in value (Intentionally).
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+console.log(Boolean(undefined));
+```
+
+The code above evaluates to...
 
 #### --distractors--
 
-Placeholder distractor 1
+`undefined`
 
 ---
 
-Placeholder distractor 2
+`true`
 
 ---
 
-Placeholder distractor 3
+`NaN`
 
 #### --answer--
 
-Placeholder answer
+`false`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+if (2 == 2) {
+  console.log("A");
+} else {
+  console.log("B");
+}
+```
+
+What does the code above evaluates to?
 
 #### --distractors--
 
-Placeholder distractor 1
+Error
 
 ---
 
-Placeholder distractor 2
+B
 
 ---
 
-Placeholder distractor 3
+No output.
 
 #### --answer--
 
-Placeholder answer
+A
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+if (2 == "2") {
+  // Statement
+}
+```
+
+What will happen when the code is ran?
 
 #### --distractors--
 
-Placeholder distractor 1
+The statement will output a warning and exits the program.
 
 ---
 
-Placeholder distractor 2
+The statement will not run.
 
 ---
 
-Placeholder distractor 3
+The statement will throw an error.
 
 #### --answer--
 
-Placeholder answer
+The statement will run.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+if (2 == "2") {
+  const a = 2;
+}
+console.log(a);
+```
+
+What does it output?
 
 #### --distractors--
 
-Placeholder distractor 1
+2
 
 ---
 
-Placeholder distractor 2
+Nothing
 
 ---
 
-Placeholder distractor 3
+`undefined`
 
 #### --answer--
 
-Placeholder answer
+Error
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you don't include `break` while implementing a switch statement?
 
 #### --distractors--
 
-Placeholder distractor 1
+The switch statement will stop abruptly.
 
 ---
 
-Placeholder distractor 2
+The switch statement will throw an error after previous statement.
 
 ---
 
-Placeholder distractor 3
+The switch statement will exit after the first match.
 
 #### --answer--
 
-Placeholder answer
+The code continues to execute the next case statement, regardless of whether the case matches.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+```js
+const a = 2;
+if (1 == "1") {
+  let b = 3;
+  console.log(a + b);
+}
+console.log(b);
+```
+
+What will be the output of this code?
+
+
```

Comment:
````suggestion
```js
const a = 2;
if (1 == "1") {
  let b = 3;
  console.log(a + b);
}
console.log(b);
```

What will be the output of this code?
````

let's not add extra lines